### ISSUE TITLE
ADDON_ACTION_FORBIDDEN - 'ElvUI' tried to call the protected function 'OnOpen()'

### ISSUE THREAD

**Issue by RealHawkBat** [tukui-org/ElvUI#1054]
### What is the issue you are having?

Note: I'm raising this ticket as the original ticket (#989) was closed, but the issues persists. I am using ElvUI v13.40 

When I access the Dungeon Journal for Aberrus raid, and I check the loot tab for the Raid, the BugSack addon detects the bug:

[ADDON_ACTION_FORBIDDEN] AddOn 'ElvUI' tried to call the protected function 'OnOpen()'.

### What is the expected behavior?

This bug should not occur and should not trigger BugSack.

Other that the error appearing in BugSack, there doesn't appear to be any other observable unexpected behavior.

### What actually happened?

The BugSack addon detects the bug:

[ADDON_ACTION_FORBIDDEN] AddOn 'ElvUI' tried to call the protected function 'OnOpen()'.

Other that the error appearing in BugSack, there doesn't appear to be any other observable unexpected behavior.

I tried again by disabling every addon I had EXCEPT ElvUI and BugSack/BugGrabber, and the error occurred.

I then disabled ElvUI so the ONLY addons I had active were BugSack/BugGrabber, and the error DID NOT occur.

### Suggested Solution/Workaround

I tried by disabling every addon I had EXCEPT ElvUI and BugSack/BugGrabber, and the error occurred.
I then disabled ElvUI so the ONLY addons I had active were BugSack/BugGrabber, and the error DID NOT error occur.

### Errors

```shell
63x [ADDON_ACTION_FORBIDDEN] AddOn 'ElvUI' tried to call the protected function 'OnOpen()'.
[string "@!BugGrabber/BugGrabber.lua"]:480: in function <!BugGrabber/BugGrabber.lua:480>
[string "=[C]"]: in function `OnOpen'
[string "@Blizzard_EncounterJournal/Blizzard_EncounterJournal.lua"]:587: in function <...zzard_EncounterJournal/Blizzard_EncounterJournal.lua:576>
[string "=[C]"]: in function `Show'
[string "@FrameXML/UIParent.lua"]:2943: in function `SetUIPanel'
[string "@FrameXML/UIParent.lua"]:2786: in function `ShowUIPanel'
[string "@FrameXML/UIParent.lua"]:2651: in function <FrameXML/UIParent.lua:2647>
[string "=[C]"]: in function `SetAttribute'
[string "@FrameXML/UIParent.lua"]:3253: in function `ShowUIPanel'
[string "@FrameXML/UIParent.lua"]:3214: in function `ToggleFrame'
[string "@FrameXML/UIParent.lua"]:1150: in function `ToggleEncounterJournal'
[string "@FrameXML/MainMenuBarMicroButtons.lua"]:1162: in function <FrameXML/MainMenuBarMicroButtons.lua:1160>
[string "=[C]"]: ?

Locals:
_ = Frame {
 RegisterEvent = <function> defined @!BugGrabber/BugGrabber.lua:486
 0 = <userdata>
 UnregisterEvent = <function> defined @!BugGrabber/BugGrabber.lua:486
 SetScript = <function> defined @!BugGrabber/BugGrabber.lua:486
}
event = "ADDON_ACTION_FORBIDDEN"
events = <table> {
 ADDON_ACTION_BLOCKED = <function> defined @!BugGrabber/BugGrabber.lua:552
 ADDON_ACTION_FORBIDDEN = <function> defined @!BugGrabber/BugGrabber.lua:552
 PLAYER_LOGIN = <function> defined @!BugGrabber/BugGrabber.lua:546
 LUA_WARNING = <function> defined @!BugGrabber/BugGrabber.lua:561
 ADDON_LOADED = <function> defined @!BugGrabber/BugGrabber.lua:506
}
```

### ElvUI Changelog & Troubleshooting

- [X] I have confirmed that I have read the [Changelog](https://github.com/tukui-org/ElvUI/blob/development/CHANGELOG.md).
- [X] I have verified ElvUI is up to date by checking /estatus.
- [X] I have verified my issue doesn't exist in the ElvUI Issue Tracker.
- [ ] I have not read step six and I am just clicking all boxes.
- [X] I have verified my issue doesn't exist in the default UI.
- [X] I have disabled all addons except the three called 'ElvUI', 'ElvUI Libraries', and 'ElvUI Options' by using `/edebug on`? This is important in order to rule out that any of your other addons are causing the issue. If you manage to pinpoint a conflict between ElvUI and another addon, make sure you include that information.

**Comment by kodewdle** [comment]
try again on dev please

**Comment by mhunk** [comment]
Just confirming the issue;

49x [ADDON_ACTION_FORBIDDEN] AddOn 'ElvUI' tried to call the protected function 'OnOpen()'.
[string "@!BugGrabber/BugGrabber.lua"]:480: in function <!BugGrabber/BugGrabber.lua:480>
[string "=[C]"]: in function `OnOpen'
[string "@Blizzard_EncounterJournal/Blizzard_EncounterJournal.lua"]:587: in function <...zzard_EncounterJournal/Blizzard_EncounterJournal.lua:576>
[string "=[C]"]: in function `Show'
[string "@FrameXML/UIParent.lua"]:2943: in function `SetUIPanel'
[string "@FrameXML/UIParent.lua"]:2786: in function `ShowUIPanel'
[string "@FrameXML/UIParent.lua"]:2651: in function <FrameXML/UIParent.lua:2647>
[string "=[C]"]: in function `SetAttribute'
[string "@FrameXML/UIParent.lua"]:3253: in function `ShowUIPanel'
[string "@FrameXML/UIParent.lua"]:3214: in function `ToggleFrame'
[string "@FrameXML/UIParent.lua"]:1150: in function `ToggleEncounterJournal'
[string "TOGGLEENCOUNTERJOURNAL"]:1: in function <[string "TOGGLEENCOUNTERJOURNAL"]:1>

Locals:
_ = Frame {
 RegisterEvent = <function> defined @!BugGrabber/BugGrabber.lua:486
 0 = <userdata>
 UnregisterEvent = <function> defined @!BugGrabber/BugGrabber.lua:486
 SetScript = <function> defined @!BugGrabber/BugGrabber.lua:486
}
event = "ADDON_ACTION_FORBIDDEN"
events = <table> {
 ADDON_ACTION_BLOCKED = <function> defined @!BugGrabber/BugGrabber.lua:552
 ADDON_ACTION_FORBIDDEN = <function> defined @!BugGrabber/BugGrabber.lua:552
 PLAYER_LOGIN = <function> defined @!BugGrabber/BugGrabber.lua:546
 LUA_WARNING = <function> defined @!BugGrabber/BugGrabber.lua:561
 ADDON_LOADED = <function> defined @!BugGrabber/BugGrabber.lua:506
}

**Comment by RealHawkBat** [comment]
> try again on dev please

Do you mean trying with the "/edebug on" command?

I have tried using the "/edebug on" command. However, as the bug is only picked up by the BugSack addon and there isn't any other observable unexpected behavior, I cannot tell if the issue pesists when using the "/edebug on" command.

**Comment by kodewdle** [comment]
i mean with the dev branch download: https://github.com/tukui-org/ElvUI/archive/refs/heads/development.zip

**Comment by kodewdle** [comment]
i believe this is fixed in 13.41, closing this issue but reopen if you have the error after `/edebug on` on 13.41 please! :)

**Comment by rifuller** [comment]
This is happening to me with edebug on.

```
4x [ADDON_ACTION_FORBIDDEN] AddOn 'ElvUI' tried to call the protected function 'OnOpen()'.
[string "@!BugGrabber/BugGrabber.lua"]:485: in function <!BugGrabber/BugGrabber.lua:485>
[string "=[C]"]: in function `OnOpen'
[string "@Blizzard_EncounterJournal/Blizzard_EncounterJournal.lua"]:586: in function <...zzard_EncounterJournal/Blizzard_EncounterJournal.lua:575>
[string "=[C]"]: in function `Show'
[string "@FrameXML/UIParent.lua"]:2968: in function `SetUIPanel'
[string "@FrameXML/UIParent.lua"]:2868: in function `ShowUIPanel'
[string "@FrameXML/UIParent.lua"]:2676: in function <FrameXML/UIParent.lua:2672>
[string "=[C]"]: in function `SetAttribute'
[string "@FrameXML/UIParent.lua"]:3278: in function `ShowUIPanel'
[string "@FrameXML/UIParent.lua"]:3239: in function `ToggleFrame'
[string "@FrameXML/UIParent.lua"]:1177: in function `ToggleEncounterJournal'
[string "TOGGLEENCOUNTERJOURNAL"]:1: in function <[string "TOGGLEENCOUNTERJOURNAL"]:1>

Locals:
_ = Frame {
 RegisterEvent = <function> defined @!BugGrabber/BugGrabber.lua:491
 0 = <userdata>
 UnregisterEvent = <function> defined @!BugGrabber/BugGrabber.lua:491
 SetScript = <function> defined @!BugGrabber/BugGrabber.lua:491
}
event = "ADDON_ACTION_FORBIDDEN"
events = <table> {
 ADDON_ACTION_BLOCKED = <function> defined @!BugGrabber/BugGrabber.lua:557
 ADDON_ACTION_FORBIDDEN = <function> defined @!BugGrabber/BugGrabber.lua:557
 PLAYER_LOGIN = <function> defined @!BugGrabber/BugGrabber.lua:551
 LUA_WARNING = <function> defined @!BugGrabber/BugGrabber.lua:566
 ADDON_LOADED = <function> defined @!BugGrabber/BugGrabber.lua:511
}
```
